src/jens/settings.py

```diff
@@ -15,26 +15,29 @@
 from configfile import CONFIG_GRAMMAR
 
 class Settings():
+    __shared_state = {}
+
     def __init__(self, logfile=None):
+        self.__dict__ = self.__shared_state
         self.logfile = logfile
```

Comment:
I think that `self.logfile` shouldn't be (re)set if `logfile` is None. If this was allowed then when the Borg was instantiated for the second time it would have a value for `logfile` that would not match the file where the logger is writing.

Example:
```python
from jens.settings import Settings
settings = Settings("jens-update")
settings.parse_config(opts.config)
# ... later on in a random method/function
settings = Settings()
print settings.logfile is None # True
```
However the logger would still be writing to $LOGDIR/jens-update.log (which is good)